### ISSUE TITLE
Add separate color swatches for "Unknown" Role

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/LinkedView#91]
In the color tab of the configuration, the Roles group has the bottommost background & border being for Other. Please insert a strong Red for "Unknown" (Which is ALWAYS an error in Role definition. Red indicates it needs to be corrected.)  You could demote "Other" to shades of yellow.

Is everything on this page mislabeled? Aren't they swatches that are for Role type color coding rather then Event type, right?

**Comment by cdhorn** [comment]
I think Unknown is basically lumped in there, I'll take a look at it though.

You are correct, I did mislabel everything there!

**Comment by emyoulation** [comment]
Thanks!  Unlumping Unknowns from the rest of the generic "Other" would help people improve their data quality.  Every Unknown role needs to be hunted down upon sighting.

**Comment by cdhorn** [comment]
Okay it ends up "Other" was "Unknown", so I relabelled it as such to be clearer.

I tweaked the colors so the same shade normally used for a divorced couple is used for very low confidence,  other misc events, unknown role, and no relationship. I think that's sort of a consistent use of it.

So I think commit eaa8764b4f3ad5498cd645659c3440b2657f8a3e closes this. Thanks!

**Comment by emyoulation** [comment]
Isn't there an "other" too?  Like Custom Roles? I have a lot of those.  "Officiant", "Celebrant", "Mourner" for the non-primary in a wedding, funeral and such. "Dependant", "Servant" & "Boarder" for others than the "Primary" (who is Head of Household) in Census.
We don't have to fix those like we do with "Unknown"s

    On Friday, December 3, 2021, 05:42:07 PM CST, Christopher Horn ***@***.***> wrote:  
 
 


Okay it ends up "Other" was "Unknown", so I relabelled it as such to be clearer.

I tweaked the colors so the same shade normally used for a divorced couple is used for very low confidence, other misc events, unknown role, and no relationship. I think that's sort of a consistent use of it.

So I think commit eaa8764 closes this. Thanks!
